data/hai missions.txt

```diff
@@ -1053,3 +1053,250 @@ mission "Hai Honeymoon"
+			`	"They can't go to our planets and experience it for themselves yet, so we thought about bringing it to them!" Vanessa says.`
+			`	"It was very hard to get enough sponsors for this, but after years of hard work we even managed to have famous art pieces be brought here, some of which date back to the european renaissance!" Xavier adds.`
+			`	Afterwards the conversation topic shifts to you as a captain, as you tell them about your journey after buying your first ship, and how you met Nanachi. You're about to order another drink for the group when a grizzled old man barges into the bar, scouting the area until he spots your group, and starts shouting: "The art convoy has been attacked! The freighter managed to escape on time but they're stranded on Prime now, the pirates are patrolling the route to the wormhole!"`
+			`	Xavier stands up, with a shocked expression. "How? We had a Leviathan escorting them!"`
+			`	After catching his breath, the man at the door responds. "The pirates must have been stealing Hai tech from ships going back home, the freighter's captain said they shredded the escort!" Chaos spreads amongst the other captains, and the group sitting with you is in anguish.`
+			choice
+				`	"Are those art pieces really that valuable?"`
+				`	"Can't you just postpone the festival until the pirates are taken care of?"`
+			`	Xavier collapses back to the table, holding his head in both hands. "Those paintings and sculptures were our main attraction! They're worth dozens of millions! It'll take a month before the Navy ever notices the pirates there and make it safe for the convoy to come here. The art exhibition is scheduled two weeks from now!"`
+			`	As they try to figure out what to do, Nanachi speaks up. "I'll go there, I pilot a human ship and if we just need to transfer cargo from one ship to the other, I could get out of there quickly, nobody would notice that a Hai ever went there."`
+			`	"Are you insane? If they managed to take down that Leviathan, you wouldn't last a second! Besides, our convoy was extremely fast, if even they were caught your chances of outrunning the pirates are next to nothing!" Vanessa says.`
+			`"Captain <last>," Nanachi turns to you. "I know you already helped me way more than anyone would ever be willing to, and I'm really grateful for that, but would you be willing to help me one last time, to get to that planet, and travel back here?" While it's certain that for a pirate fleet to take down a Leviathan so easily they must be incredibly powerful, and this could be extremely dangerous, you're sure that if Nanachi tries to get to Prime by herself she'll just get herself killed.`
```

Comment:
`way more` is something children say; maybe replace this with something else. Also this is a huge run-on sentence and should be split up.